src/NRedisStack/Graph/GraphCommands.cs

```diff
@@ -81,7 +80,9 @@ public ResultSet CallProcedure(string graphName, string procedure, IEnumerable<s
                 queryBody.Append(string.Join(",", kwargsList));
```

Comment:
I also can't think of other options besides YEILD,
Maybe in the past there were more options and the code has remained that way ever since.
What would you like to change it to?

Reply:
Even the lookup on the very specific key `y` seems tribal :)
If `YIELD` is the only possible option, it seems a bit much having to define a dictionary that contains a single key `y`.
So, in the spirit of the other arguments, a list of output parameters would make sense, and if it contains any values, then we add `YIELD` to the query.

Otherwise, if we were to keep the dictionary approach as-is, then the mapping could be `key => value`, where `key` is name of a clause, i.e., `YIELD`, and the `value` is a list of parameters, i.e., `val0, val1, val2.
However, I am not sure what other clauses there are available in a CALL procedure query.

Reply:
After consulting with @chayim, I think we'll leave it as it is for now.
thanks anyway!